app/views/candidates/_form.html.erb

```diff
@@ -0,0 +1,98 @@
+
+    <div class="my-1">
+      <div class="form-label fw-bold"><%= form.label :phone_number %></div>
+      <%= form.text_field :phone_number, required: true %>
+      <% form.object.errors.full_messages_for(:phone_number).each do |message| %>
+        <div class="error"><%= message %></div>
+    <% end %>
+    </div>
+
+    <div class="my-1">
+      <div class="form-label fw-bold"><%= form.label :university %></div>
+      <%= form.text_field :university, required: true %>
+      <% form.object.errors.full_messages_for(:university).each do |message| %>
+        <div class="error"><%= message %></div>
+      <% end %>
+    </div>
+
+    <div class="my-1">
+      <div class="form-label fw-bold"><%= form.label :gpa %></div>
+      <%= form.text_field :gpa, required: true %>
+      <% form.object.errors.full_messages_for(:gpa).each do |message| %>
+        <div class="error"><%= message %></div>
+      <% end %>
+    </div>
+
+    <div class="my-1">
+      <div class="form-label fw-bold"><%= form.label :bio %></div>
+      <%= form.text_area :bio, size: "100x5", required: true %>
+      <% form.object.errors.full_messages_for(:bio).each do |message| %>
+        <div class="error"><%= message %></div>
+      <% end %>
+    </div>
+
+    <div class="my-1">
+      <div class="form-label fw-bold"><%= form.label :picture %></div>
+      <%= form.file_field :picture, required: true %>
+      <% form.object.errors.full_messages_for(:picture).each do |message| %>
+        <div class="error"><%= message %></div>
+      <% end %>
+    </div>
+
+    <div class="my-1">
+      <div class="form-label fw-bold"><%= form.label :cv %></div>
+      <div><%= form.file_field :cv, required: true %></div>
+      <% form.object.errors.full_messages_for(:cv).each do |message| %>
+        <div class="error"><%= message %></div>
+      <% end %>
+    </div>
+
+    <div class="my-1">
+      <div class="form-label fw-bold"><%= form.label :docs %></div>
+      <div><%= form.file_field :docs, :multiple => true, required: true %></div>
+      <% form.object.errors.full_messages_for(:docs).each do |message| %>
+        <div class="error"><%= message %></div>
+      <% end %>
+    </div>
+
+    <% if candidate.persisted? %>
+      <div class="my-1">
+        <div class="form-label fw-bold"><%= form.label :status %></div>
+        <%= form.select :status, %w[applied test interview1 interview2 interview3 hired], selected: 'applied', required: true %>
+        <% form.object.errors.full_messages_for(:status).each do |message| %>
+          <div class="error"><%= message %></div>
+        <% end %>
+      </div>
+    <% else %>
+      <div class="my-1">
+        <div class="form-label fw-bold"><%= form.label :status %></div>
+        <%= form.select :status,['applied'] , selected: 'applied', required: true %>
+        <% form.object.errors.full_messages_for(:status).each do |message| %>
+          <div class="error"><%= message %></div>
+        <% end %>
+      </div>
+    <% end %>
```

Comment:
code is repeating here you can set a default status in this case here

Reply:
done, set default status = applied for enum